playx-cdi/src/main/resources/META-INF/openwebbeans/openwebbeans.properties

```diff
@@ -5,7 +5,7 @@ org.apache.webbeans.scanExclusionPaths=\
 /aeron,\
 /aether-,\
 /agrona,\
-/akka-,\
```

Comment:
it can stay, does not hur but can be good to move pekko exclusion (IIRC it was sorted)